src/components/Identity/index.js

```diff
@@ -100,6 +149,12 @@ const createIdentity = ({ config, logger, cookie }) => {
     commands: {
       getEcid() {
         return optIn.whenOptedIn().then(getEcid);
+      },
+      setCustomerIDs(options) {
```

Comment:
Should we simplify this command to `setCustomerIds(ids)`?

This way customers will do:

```
alloy("setCustomerIds", {
    email: {
    },
    crm: {
    }
});
```
instead of:

```
alloy("setCustomerIds", {
    customerIds: {
        email: { },
        crm: {}
    }
});
```
@jonsnyder  @Aaronius